packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -269,6 +270,16 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
           },
         },
 
+        /**
+         * This modifier is necessary to retain behaviour from popper v1 where untethered poppers are allowed by
+         * default. i.e. popper is still rendered fully in the viewport even if anchor element is no longer in the
+         * viewport.
+         */
+        unstable_disableTether && {
```

Comment:
If `unstable_disableTether` is false wouldn't this be adding `false` as a value into the array? @ling1726

Reply:
saw this when I was checking old PR on how to do popper fix.
There's a `.filter(Boolean)` following this array of modfiers, that is hidden from the github review view :)